docroot/modules/custom/civictheme_styleguide/components/video_player.inc

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * @file
+ * Video Player component styleguide integration.
+ */
+
+use Drupal\civictheme_styleguide\Styleguide;
+use Drupal\cs_generated_content\CsGeneratedContentHelper;
+
+/**
+ * Implements _civictheme_styleguide_component_NAME().
+ */
+function _civictheme_styleguide_component_video_player() {
+  $items = [];
+
+  /** @var \Drupal\cs_generated_content\CsGeneratedContentHelper $helper */
+  $helper = CsGeneratedContentHelper::getInstance();
+  $variations = [
+    [
+      'title' => 'Video Player component, Light',
+      'components' => [
+        [
+          'type' => 'civictheme_video',
+          'name' => $helper::staticName(6),
+          'transcript_link' => $helper::randomUrl(),
+          'video_file' => $helper::staticFile('mp4'),
```

Comment:
Not able to generate staticFile('mp4') 
only images get generated.

Reply:
@joshua-salsadigital 
can you use a fixture mp4 file - store it in `civictheme_styleguide/assets/` dir. You can use a video in the CivicTheme library - please copy that file